server/routers/payment.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const app = express();
+const router = express.Router();
+const stripe = require('../routers/constants/stripe');
+app.set("view engine", "pug");
+app.use(require("body-parser").urlencoded({extended: false}));
```

Comment:
Intresting. I would need more time to really understand the complete breakdown of this.